client/src/components/users/types.ts

```diff
@@ -51,9 +61,42 @@ export class UserCollection {
     return this.users.filter((user) => user.isSelected && user.isRevoked()).length;
   }
 
-  sort(_property: SortProperty, _ascending: boolean): void {
-    this.users.sort((_item1, _item2) => {
-      return 1;
+  sort(property: SortProperty, ascending: boolean): void {
```

Comment:
`SortProperty` also contains `Email` but it's not used in the code. Either add the ability to sort on it or remove it from the enum.

This function could probably benefit from using switch/cases instead of ifs, the linter should be able to warn us if you forget to cover a case (like here with the `Email`).